### ISSUE TITLE
theres already a text view in the VC and its misleading the tests

### ISSUE THREAD

**Issue by tomalley104** [learn-co-curriculum/tasks-Today#2]
so theres already a textView in the eventDetail VC with the right accessibility identifier for tests. It's plain white, so  students arent realizing its there, and then when they run the tests they get a FAIL because tests are reading from this blank textView. 

SO one of three things: 
1. make it more explicit that there IS a text view there in the reading
2. make the textView a different color
3. remove it and update the instructions to say "add a textview"
